### PR TITLE
SquBounce Animated Art

### PR DESCRIPTION
**Pull request by sanusisusi**

### CODE REVIEW

**Comment by sophiabrandt** [comment]
@sanusisusi please add the necessary changes to the file `include.js`. Instructions are in the [README](./README.md) file.

**Comment by sanusisusi** [comment]
> Mem

**Comment by sanusisusi** [comment]
@sophiabrandt i think i have done that, i changed the include.js file

**Comment by sophiabrandt** [comment]
@sanusisusi please take a look at your [pull request files](https://github.com/zero-to-mastery/Animation-Nation/pull/1785/files), there are no changes to the file `include.js`.

**Comment by sanusisusi** [comment]
@sophiabrandt Done right, sorry it was only on my repo without pushing

File: include.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ let cards = [
     author: 'Ibaad',
     githubLink: 'https://github.com/ibaaddurrani'
   }
```
Comment (sophiabrandt) [inline include.js:15]:
```suggestion
  },
```
There needs to be a comma, otherwise it's not a valid Javascript array.

**Pull request merged**